src/js/apis/c2c/TagsService.js

```diff
@@ -0,0 +1,30 @@
+function TagsService(api) {
+  this.api = api;
+}
+
+TagsService.prototype.add = function (document) {
+  if (document.type !== 'r') {
+    // As for now, only routes may be tagged.
+    return Promise.reject(new Error('Tags are not supported for this type of documents'));
+  }
+
+  const data = {
+    document_id: document.document_id,
+  };
+
+  return this.api.post('/tags/add', data);
+};
```

Comment:
I would propose using object destructuring

```suggestion
TagsService.prototype.add = function ({type, document_id}) {
  if (type !== 'r') {
    // As for now, only routes may be tagged.
    return Promise.reject(new Error('Tags are not supported for this type of documents'));
  }
  return this.api.post('/tags/add', { document_id });
};
```